src/AppBundle/Form/PostType.php

```diff
@@ -32,14 +32,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         // for the full reference of options defined by each form field type
         // see http://symfony.com/doc/current/reference/forms/types.html
         $builder
-            ->add('title')
-            ->add('summary', 'textarea')
+            ->add('title', null, ['label' => 'label.title'])
```

Comment:
this is not compatible with php 5.3 (as stated in composer.json). so I'd suggest increasing the php requirement for the demo. we do not need to support 5.3.